openApiDocs/beta/Bookings.yml

```diff
@@ -2515,10 +2515,10 @@ components:
       properties:
```

Comment:
Could you undo the last commit that added the Open API descriptions? These descriptions are old and unrelated to the PR.